.github/workflows/changelog_reminder.yaml

```
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+    types: [opened, ready_for_review]
```

How about running the reminder once a review is being submitted?
```suggestion
  pull_request_review:
    types: [submitted]
```